OpenRA.Mods.D2k/Traits/SpiceBloom.cs

```diff
@@ -34,6 +37,9 @@ public class SpiceBloomInfo : ITraitInfo, Requires<RenderSpritesInfo>
 
 		public readonly string ResourceType = "Spice";
 
+		[Desc("Spice blooms only grow on this terrain type.")]
+		public readonly string GrowthTerrainType = null;
+
 		[Desc("The weapon to use for spice creation.")]
 		[WeaponReference]
 		public readonly string Weapon = "SpiceExplosion";
```

Comment:
Can we default this to null so that you don't need to override it with an empty entry in the spawnpoint?